.php_cs

```diff
@@ -9,6 +9,17 @@
  * with this source code in the file LICENSE.
  */
 
+$header = <<<EOF
+This file is part of Gush package.
+
+(c) Luis Cordova <cordoval@gmail.com>
+
+This source file is subject to the MIT license that is bundled
+with this source code in the file LICENSE.
+EOF;
+
+Symfony\CS\Fixer\Contrib\HeaderCommentFixer::setHeader($header);
```

Comment:
oh interesting, but you know there is also a command in gush to fix this too right?